src/main/java/athleticli/commands/sleep/AddSleepCommand.java

```diff
@@ -1,4 +1,35 @@
 package athleticli.commands.sleep;
 
-public class AddSleepCommand {
+import athleticli.commands.Command;
+import athleticli.data.Data;
+
+import athleticli.data.sleep.Sleep;
+import athleticli.data.sleep.SleepList;
+
+public class AddSleepCommand extends Command {
+
+    private String from;
+    private String to;
+
+    public AddSleepCommand(String from, String to) {
+        this.from = from;
+        this.to = to;
+    }
```

Comment:
In activity,  we used `LocalDateTime` for datetime. Should we use `LocalDateTime` for all our datetime tracking to be consistent?

Reply:
Yes will be implemented on the next iteration